Functions/xASL_bids_PrepareDRO.m

```diff
@@ -0,0 +1,96 @@
+function xASL_bids_PrepareDRO(droTestPatient,droSubject)
```

Comment:
Can you change the name to xASL_test_PrepareDROTest or something like this - it is not preparing DRO, but preparing a DRO test, right? So that's more about testing than about BIDS.

Reply:
So my idea was to use the output of the asldro python script exactly like it is and then change some file names (add `sub-...` etc. in `lines 45-55`) and add all necessary sidecars & missing fields (dataset description in `lines 61-74`) so that the `BIDS2Legacy` script can convert it to something that can be read by **ExploreASL**.
Having a script for this could be quite useful, since we could test other parts of **ExploreASL** with the dro data as well.
I'm fine with renaming and/or improving the script in general and it will be used almost exclusively for testing, but not just for this individual test, right?

Reply:
I see, I get the general idea - and I completely agree. Do I get it right that what you do here is fixing all the small BIDS issues for DRO? i.e. things that Aaron should fix in the future because DRO output should be (in theory and in future) in BIDS?

We should then adapt the name of the function to something like xASL_bids_DRO2BIDS. And this function should become obsolete once Aaron fixes everything, shouldn't it?

Reply:
Exactly that was the idea. I just wasn't sure if that's Aaron making mistakes or just me not using it correctly (but I set everything to default). And it is correct that the files have to start with `sub-Sub1...` right?
I'll change the name to `xASL_bids_DRO2BIDS` for now, sound way much better 👍

Reply:
Fixed in 9f0a3ab.